### PR TITLE
Mtoledo/add interruptible parameter

### PR DESCRIPTION
**Pull request by migueltol22**

### CODE REVIEW

**Comment by matthewphsmith** [comment]
Are we going to expose this configuration to the user?

**Comment by migueltol22** [comment]
> Are we going to expose this configuration to the user?

Yes, the idea is that this can be set at the workflow or task level.

**Comment by matthewphsmith** [comment]
This part looks good, do you want to do that plumbing in the same PR? Or make those changes separately?

**Comment by migueltol22** [comment]
> This part looks good, do you want to do that plumbing in the same PR? Or make those changes separately?

I can do those changes here as well. Was going to sync with Yee to test end to end today and then will iterate off of that.

File: flytekit/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 from __future__ import absolute_import
 import flytekit.plugins
 
-__version__ = '0.5.1b3'
+__version__ = '0.5.1b4'
```
Comment (wild-endeavor) [inline flytekit/__init__.py]: bump again, already past this on master.

Comment (wild-endeavor) [reply flytekit/__init__.py]: one more time.

**Pull request merged**